kvbc/src/merkle_tree_db_adapter.cpp

```diff
@@ -387,6 +397,14 @@ SetOfKeyValuePairs DBAdapter::lastReachableBlockDbUpdates(const SetOfKeyValuePai
 
   for (const auto &[k, v] : non_provable_kvs) {
     dbUpdates[DBKeyManipulator::genNonProvableBlockDbKey(blockId, k)] = v;
+    if (blockId) {
+      const auto &stale_kv = getValueForNonProvableKey(k, blockId - 1);
```

Comment:
Nit: I think that even though the style for this file is incorrect (camelCase), it might be better to stick to it (i.e. `staleKv`) and then change all occurrences it in one go. Applies to other uses too.

Reply:
done